source/geant4.f90

```diff
@@ -0,0 +1,145 @@
+module geant4
+
+  use floatPrecision
+  use crcoall
+  use iso_c_binding, only: C_NULL_CHAR
+  use numerical_constants
+
```

Comment:
Don't include other modules in the module header unless they are strictly needed there as the content is bleeding through to code using this module. Only floatPrecision is needed here.

Reply:
well considering lout, lerr, zero are all in those modules.
The C_NULL_CHAR was from stuff long ago, so that can go yes.